integration_files/SiFive_E21/sim.py

```diff
@@ -147,7 +147,7 @@ def trace_log(raw_rtl_log,dump, rtl_log):
                         f = open(rtl_log, "a")
                         a = "{0}\t\t{1}\t\t{2}\t\t{3}\t\t{4}\t\t\t{5}\t{6}\t{7}\n"
                         b = "{0}\t\t{1}\t\t{2}\t\t{3}\t\t{4}\t\t{5}\t{6}\n"
-                        opcode = ["beq","bne","blt","bge","btyu","c.beqz","bgeu","sb","sd","sh","c.sw","fence","fence.i","ecall","break"]
+                        opcode = ["beq","bne","blt","bge","bltu","c.beqz","bgeu","sb","sd","c.sw","c.swsp","sh","fence","fence.i","ecall","break"]
```

Comment:
This list of opcodes is based on a Base ISA and a very small number of ISA extensions. Need to extend this list in the future as more instructions are added to it.